lib/experimental/subject.rb

```diff
@@ -12,8 +12,8 @@ def experiment_bucket(name)
       Experimental.source[name].try { |e| e.in?(self) ? e.bucket(self) : nil }
     end
 
-    def in_bucket?(name, bucket)
-      in_experiment?(name) && experiment_bucket(name) == bucket
+    def in_bucket?(name, bucket, *args)
```

Comment:
@key88sf I like the idea of a plural method a bit more than having a param of 2 types.  @kulte thoughts?